quickstart/hono_commands_sb.py

```diff
@@ -0,0 +1,283 @@
+
+        event.sender.send(msg)
+        event.sender.close()
+        event.connection.close()
+        print('[sent]')
+
+
+class EventsHandler(MessagingHandler):
+    def __init__(self, server, receiver_address, tenant_id):
+        super(EventsHandler, self).__init__()
+        self.server = server
+        self.receiver_address = receiver_address
+        self.tenant_id = tenant_id
+
+    def on_start(self, event):
+        conn = event.container.connect(self.server, user="consumer@HONO", password="verysecret")
+        event.container.create_receiver(conn, source=self.receiver_address, options=[AtLeastOnce()])
+        print('[events handler connected for tenant {}]'.format(self.tenant_id))
+
+    def on_message(self, event):
+        if event.message.body is not None:
+            body = json.loads(event.message.body)
+            if body["topic"].split("/")[-1] == "request":
+                print('[request event received]')
+                print(json.dumps(body, indent=2))
+                request_id = body["value"]["requestId"]
+                Container(CommandsInvoker(uri, address, request_id=request_id)).run()
+                event.receiver.close()
+                event.connection.close()
+
+            elif body["topic"].split("/")[0] == new_tenant_id:
+                print('[event received from new tenant [{}]]'.format(new_tenant_id))
+                print('[remove new created resources]')
+                headers = {
+                    'Content-Type': 'application/json',
+                }
+
+                response = requests.delete(
+                    '{}/v1/devices/{}/{}'.format(hono_ep, new_tenant_id, new_device_id), headers=headers)
+
+                print("Deleting device [{}] with code [{}]".format(new_device_id, response.status_code))
+
+                response = requests.delete(
+                    '{}/v1/credentials/{}/{}'.format(hono_ep, new_tenant_id, new_device_id), headers=headers)
+
+                print("Delete credentials [{}]".format(response.status_code))
+
+                response = requests.delete(
+                    '{}/v1/tenants/{}'.format(hono_ep, new_tenant_id), headers=headers)
+
+                print("Deleting tenant [{}] with code[{}]".format(new_tenant_id, response.status_code))
+
+                event.receiver.close()
+                event.connection.close()
+
+    def on_connection_closed(self, event):
+        print("[events handler connection for tenant {} closed]".format(self.tenant_id))
+        if self.tenant_id == new_tenant_id:
+            os.kill(os.getpid(), signal.SIGINT)
+
+
+# Parse command line args
+options, reminder = getopt.getopt(sys.argv[1:], 't:d:p:')
+opts_dict = dict(options)
+tenant_id = os.environ.get("TENANT") or opts_dict['-t']
+device_id = os.environ.get("DEVICE_ID") or opts_dict['-d']
+new_password = os.environ.get("PASSWORD") or opts_dict['-p']
```

Comment:
The command arguments must have greater priority than environment variables.
```suggestion
tenant_id = opts_dict['-t'] or os.environ.get("TENANT")
device_id = opts_dict['-d'] or os.environ.get("DEVICE_ID")
new_password = opts_dict['-p'] or os.environ.get("PASSWORD")
```